src/components/modebar/buttons.js

```diff
@@ -134,6 +134,58 @@ modeBarButtons.lasso2d = {
     click: handleCartesian
 };
 
+modeBarButtons.drawclosedpath = {
+    name: 'drawclosedpath',
+    title: function(gd) { return _(gd, 'Draw Closed Freeform'); },
```

Comment:
```suggestion
    title: function(gd) { return _(gd, 'Draw closed freeform'); },
```

Not a big deal, but I think a single initial cap makes more sense for all of the new buttons. That's how we capitalize all the other button titles, except `Edit in Chart Studio` (makes sense as Chart Studio is a proper name) and `Box Select` and `Lasso Select` - and I guess we should change those two at some point but unfortunately that would require editing all the translation files that already use these... so let's leave them alone for now and just change the new ones in this PR.

Reply:
Good call. Updated in https://github.com/plotly/plotly.js/pull/4775/commits/44dec73c5f682ef97454ea86a31c0ff4eb30d11a.